module/IlluminaBeadArrayFiles.py

```diff
@@ -576,8 +576,8 @@ def normalize_intensities(self, x, y, threshold = True):
         tempx3 = tempx2 - self.shear * tempy2
         tempy3 = tempy2
 
-        xn = tempx3 / self.scale_x
-        yn = tempy3 / self.scale_y
+        xn = tempx3 / self.scale_x if self.scale_x != 0 else nan
```

Comment:
Upon further consideration, this should really return Inf or -Inf, depending on the value of tempx3 (or NaN if tempx3 is Nan). This would be most consistent with the values displayed in GenomeStudio, and probably makes more sense. The better option might be to switch over to the numpy float64 type to get better standard behavior.  I'll have to look into whether that will break anything else.